roles/slurm-perf/tasks/packages-redhat.yml

```diff
@@ -0,0 +1,54 @@
+---
+- name: yum repos
```

Comment:
Given redhat is not yet a supported OS for this role, do we want to remove this file for now?

Reply:
Yes